x/datapool/README.md

```diff
@@ -1,4 +1,5 @@
 ## Deploy and instantiate NFT smart contract
+## Deploy and instantiate NFT smart contract
```

Comment:
This comment was duplicate.

Reply:
Oops, thank you

Reply:
fixed in [e5d8c20](https://github.com/medibloc/panacea-core/pull/306/commits/e5d8c2039353b0cbd51d364469ca6f37e5359ce7)